scripts/generate_templates.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+# This file is part of ap_verify_dataset_template.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Script for automatically generating differencing templates for this dataset.
+# It takes roughly <TBD> hours to run on rubin-devl.
```

Comment:
For `ap_verify_ci_hits2015`, and especially `ap_verify_hits2015`, I found it very useful to know how long the script actually takes to run so that I could plan accordingly. In those packages, it was 5 hours and 24 hours respectively, so "a number of" wouldn't have been very helpful.